lib/Runtime/Base/ThreadContext.h

```diff
@@ -636,6 +636,7 @@ class ThreadContext sealed :
     StackProber * stackProber;
     bool isThreadBound;
     bool hasThrownPendingException;
+    bool * hasBailedOutBitOffset;
```

Comment:
Nit: This naming is slightly off, I feel, as this isn't really an offset, but is a full-fledged pointer. This becomes more of an issue in cases like https://github.com/Microsoft/ChakraCore/pull/4535/files#diff-b1ac070605da21777f9c4d31b69a2014R162, where you're setting it with a pointer plus an _actual_ offset.

Reply:
Done.